src/main/java/uk/gov/justice/hmpps/prison/api/model/OffenderKeyDates.java

```diff
@@ -0,0 +1,29 @@
+package uk.gov.justice.hmpps.prison.api.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.time.LocalDate;
+
+@ApiModel(description = "Offender Key Dates")
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@Data
+public class OffenderKeyDates {
+
```

Comment:
We (CRD) currently calculate more release dates than these three, i.e. we currently cater for the following
CRD,
LED,
SED,
NPD,
ARD,
TUSED,
PED,
HDCED,
SLED (This is the SED and LED when they are both the same)

However this is being added to - will provide a full list separately of all the types of key dates